code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -111,5 +111,6 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
 	/obj/item/reagent_containers/food/drinks/bottle/homemaderum = 1,
 	/obj/item/reagent_containers/food/snacks/grown/poppy = 10,
 	/obj/item/throwing_star = 1,
+	/obj/machinery/allaccess = 2
```

Comment:
Spawn the circuit instead of the machine

Reply:
also lacks a comma at the end

Reply:
thats probably why it didnt compile